site/pages/react/react-hooks.mdx

```diff
@@ -37,6 +37,40 @@ export default function MyLoginPage() {
 }
 ```
 
+### Email authentication with redirect params
+
+```tsx twoslash
+import React from "react";
+import {
+  type UseAuthenticateResult,
+  useAuthenticate,
+} from "@account-kit/react";
+
+// This examples uses email authentication
+// you can also use passkeys if the user has one created
+export default function MyLoginPage() {
+  const { authenticate, isPending } = useAuthenticate();
+  const [email, setEmail] = React.useState("");
+
+  const handleAuthenticate = () => {
+    const redirectParams = new URLSearchParams();
+    redirectParams.set("myParam", "someValue");
+    authenticate({ type: "email", email, redirectParams });
```

Comment:
In examples, let's always explicitly provide a `emailMode` param, as in `emailMode: "magicLink"` here. While `emailMode` is optional and defaults to `"magicLink"`, this is done purely for backwards compatibility reasons. Since our goal is to direct people towards `"otp"`, we should downplay that magic link is the default.